src/MallardMessageHandlers/IsExternalInit.cs

```diff
@@ -0,0 +1,5 @@
+﻿namespace System.Runtime.CompilerServices;
+
+internal static class IsExternalInit
```

Comment:
Not sure the purpose of this class? can it be commented on what this do?

Reply:
It's to be able to use `record` (in MemorySimpleCacheService.cs).

Reply:
I added a comment.